src/frontend/src/components/public/PIAFormTabs/intake/index.tsx

```diff
@@ -270,7 +270,7 @@ export const PIAFormIntake = () => {
                   name="pi-options-radio"
                   value={option.key}
                   onChange={handlePIOptionChange}
-                  checked={option.value === pia?.hasAddedPiToDataElements}
```

Comment:
just one quick question, does the three = not work?

Reply:
apparently it is not effective against 'null'

Reply:
<img width="139" alt="image" src="https://user-images.githubusercontent.com/87394256/223784244-d60cfb6b-edca-4ceb-88c4-ae413160452a.png">

Reply:
I can try and pull your branch to understand better to what you're referring to

Reply:
You're comparing undefined and null. Hence it is giving you false for === and true with ==.
<img width="186" alt="image" src="https://user-images.githubusercontent.com/87394256/223786769-d90897ad-49c6-4753-a8b9-1ab60b77d4d8.png">